web/src/main/java/org/akaza/openclinica/service/ImportServiceImpl.java

```diff
@@ -1306,6 +1346,15 @@ private Object validateItem(ImportItemDataBean itemDataBean, CrfBean crf, EventC
                 return new ErrorObj(NO_CHANGE, null);
 
             } else {
+            	if(isEventCrfCompleted(eventCrf)) {            		
+            		String message = "import XML";
+            		String detailedNotes = "Update via Import";
```

Comment:
Create static final variables at the top of the class for hard coded values so that can update them easily when needed.

Reply:
did code changes based on code review